### DISCUSSION TITLE
www.google.com周期性打不开，症状比较奇怪，可能跟dns没关系

### DISCUSSION THREAD

**Discussion by crysislinux** [IrineSistiana/mosdns#328] [Q&A] [answered]
浏览器Chrome

具体表现是：
1. www.google.com 一开始能打开
2. 一段时间后突然打不看，看chrome dev tools里network，域名没解析出地址，但是浏览器也并不报错，就是一直卡着
3. 如果不管，卡很长时间后（> 10分钟），卡住的网页又打开了
4. www.google.com 又能正常使用一段时间

当 www.google.com 打不开时：
1. 别的Google域名能打开，比如 www.google.ca , www.google.com.hk ，其他被墙网站也能打开
2. 在隐身窗口中能打开 www.google.com
3. 重启chrome能打开 www.google.com
4. 在别的浏览器中能打开 www.google.com
5. 在终端中curl能打开 www.google.com

已尝试的办法:
1. 关闭chrome的secure dns
2. 建立本地的doh，在chrome 的secure dns里用本地的doh

目前在尝试一直使用 www.google.ca 和 关闭 quic，不知道能不能奏效

不知道有没有人有同样的问题。之前有另外一个奇怪的问题在 https://github.com/IrineSistiana/mosdns/discussions/294 里找到了答案，不知道这次这个能不能解决 😂😂，多谢

**Comment by IrineSistiana** [comment, marked as answer]
算命忙猜:

> [www.google.com](http://www.google.com/) 一开始能打开。一段时间后突然打不看。

像是连接复用的问题。连接已经出问题了。但是 chrome 并不知道，还在用。

>如果不管，卡很长时间后（> 10分钟），卡住的网页又打开了

这条连接长时间空闲没数据，被 chrome 里的定时器关了。chrome 用新连接连网页就开了。


> 在隐身窗口中能打开 [www.google.com](http://www.google.com/)
> 重启chrome能打开 [www.google.com](http://www.google.com/)

重启和新隐身窗口都会强制 chrome 用只用新连接。

> 其他被墙网站也能打开

只有 google 用了 quic。

quic 的锅。

**Reply by crysislinux** [reply]
可能真是这个原因，把443端口的udp流量drop了好像没问题了，观察几天看看，多谢

**Reply by crysislinux** [reply]
用了半天没出问题，看来真是quic的毛病。
